scripts/cmake/packaging/Pack.cmake

```diff
@@ -126,3 +126,6 @@ endif()
 
 configure_file(Documentation/README.txt.in ${PROJECT_BINARY_DIR}/README.txt)
 install(FILES ${PROJECT_BINARY_DIR}/README.txt DESTINATION .)
+
+install(FILES ${PROJECT_BINARY_DIR}/CMakeCache.txt DESTINATION ${CMAKE_INSTALL_INFODIR})
```

Comment:
For reproducible builds? Nice.

Reply:
Not for reproducing but for checking which config one actually has..